### PR TITLE
Modify index.html and add images.html, projects.html. 

### PR DESCRIPTION
**Pull request by yjzhang111**
Modified index.html to turn the template into my own portfolio. Added a gallery page and a project page on top of the main portfolio page.

### CODE REVIEW

File: portfolio/src/main/webapp/images.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Favourite Places</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <a href="index.html" class="button">Home</button>
+    <a href="https://www.linkedin.com/in/yijia-sylvia-zhang" class="button">LinkedIn</a>
+	<a href="https://github.com/yjzhang111" class="button">GitHub</a>
```
Comment (albert-g3) [inline portfolio/src/main/webapp/images.html:12]: nit: appears formatting is a little off here, is there a way to have this auto format?

**Review by albert-g3 — APPROVED** [review]
well done!
Just one comment about formatting the files, generally there is an automated way to do that through the editor or some script, but if you don't have either then don't worry about it.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains the Yijia's portfolio and STEP projects.
```
Comment (mutisog) [inline README.md:3]: nit: can remove "the".

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,39 @@
...
+        <p>Growing up, I have lived in three different countries. And I still have a great interest in exploring other parts of the world. Hopefully I can go study abroad after the global pandemic.</p>
+        <p>I am minoring in music, and I hope to learn more about music production. I love playing music instruments. Some of the instruments I play include</p>
+        <ul>
+            <li>Piano</li>
+            <li>Acoustic/Classical Guitar</li>
+            <li>Harmonica</li>
+            <li>Drums</li>
+        </ul>
+        <p>In my freetime, I watch a lot of Japanese anime and drama. Below is a poster from my favourite studio.</p>
+        <img src="images/weathering_with_you.jpg" alt="The poster of Weathering With You: a girl standing and a boy sitting on the ground, both looking at the sky" style="width:500px;height:333px;">
```
Comment (mutisog) [inline portfolio/src/main/webapp/index.html:33]: Kudos on adding alt text! Good practice :)

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,22 @@
...
+      if I happen to meet you, one day in the future, by then, \
+      I want to become someone you can be proud to know.', 
+      'I probably just want to leave a trace of myself behind in this world.', 
+      'It must really be a lonelier journey than anyone could imagine. \
+      Cutting through absolute darkness, \
+      encountering nothing but the occasional hydrogen atom. \
+      Flying blindly into the abyss, \
+      believing therein lie the answers to the mysteries of the universe.'];
 
   // Pick a random greeting.
```
Comment (mutisog) [inline portfolio/src/main/webapp/script.js:30]: nit: "greeting" --> "quote"

**Review by mutisog — COMMENTED** [review]
Looking good! Left a few nits (minor comments), otherwise lgtm

**Review by mutisog — APPROVED** [review]

**Pull request merged**